app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,76 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+    <%= t('doc_auth.headings.combined_upload') %>
```

Comment:
Nit: Can we indent this file by 2 spaces (rather than 4), which is the standard indentation?

Reply:
Hmm, weird. I did the autocorrect option for erb lint. I'll come back and fix this last.

Reply:
Unfortunately I don't think ERBLint does much for us regarding whitespace outside of `<%` `%>` tags